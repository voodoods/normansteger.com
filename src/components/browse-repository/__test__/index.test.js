import React from 'react'
import { shallow } from 'enzyme'
import BrowseRepository from '../'

describe('<BrowseRepository /> component index', () => {
  it('expects component to match snapshot', () => {
    const wrapper = shallow(<BrowseRepository />).dive()
    expect(wrapper).toMatchSnapshot()
  })
})
