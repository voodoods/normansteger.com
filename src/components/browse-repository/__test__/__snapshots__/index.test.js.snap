// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BrowseRepository /> component index expects component to match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <BrowseRepository
    openInNewTab={[Function]}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <div
        className="pure-u-1 container"
      >
        <div
          className="pure-u-sm-1 pure-u-md-1-4 block-left"
        >
          <i
            className="fa fa-code-fork fa-3x "
          />
        </div>
        <div
          className="pure-u-sm-1 pure-u-md-3-4 block-right "
        >
          <b>
            Want to take a look into the code?
          </b>
          <br />
          <span>
            Browse the repository of this site!
          </span>
        </div>
      </div>,
      "className": "BrowseRepository pure-u-1",
      "onClick": [Function],
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="pure-u-sm-1 pure-u-md-1-4 block-left"
          >
            <i
              className="fa fa-code-fork fa-3x "
            />
          </div>,
          <div
            className="pure-u-sm-1 pure-u-md-3-4 block-right "
          >
            <b>
              Want to take a look into the code?
            </b>
            <br />
            <span>
              Browse the repository of this site!
            </span>
          </div>,
        ],
        "className": "pure-u-1 container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <i
              className="fa fa-code-fork fa-3x "
            />,
            "className": "pure-u-sm-1 pure-u-md-1-4 block-left",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "fa fa-code-fork fa-3x ",
            },
            "ref": null,
            "rendered": null,
            "type": "i",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <b>
                Want to take a look into the code?
              </b>,
              <br />,
              <span>
                Browse the repository of this site!
              </span>,
            ],
            "className": "pure-u-sm-1 pure-u-md-3-4 block-right ",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Want to take a look into the code?",
              },
              "ref": null,
              "rendered": "Want to take a look into the code?",
              "type": "b",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Browse the repository of this site!",
              },
              "ref": null,
              "rendered": "Browse the repository of this site!",
              "type": "span",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  font-size: 0.8em;
  cursor: pointer;

  b,
  span,
  i {
    color: #ccc;
    line-height: 1.6em;
  }

  .block-right {
    margin: 12px 0;
  }

  &:hover {
    b,
    span,
    i {
      color: #000;
    }
  }

  @media (max-width: 768px) {
    .container {
      width: 50%;
      margin: 0 25%;
    }

    .block-left,
    .block-right {
      text-align: center;
    }
  }
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <div
          className="pure-u-1 container"
        >
          <div
            className="pure-u-sm-1 pure-u-md-1-4 block-left"
          >
            <i
              className="fa fa-code-fork fa-3x "
            />
          </div>
          <div
            className="pure-u-sm-1 pure-u-md-3-4 block-right "
          >
            <b>
              Want to take a look into the code?
            </b>
            <br />
            <span>
              Browse the repository of this site!
            </span>
          </div>
        </div>,
        "className": "BrowseRepository pure-u-1",
        "onClick": [Function],
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="pure-u-sm-1 pure-u-md-1-4 block-left"
            >
              <i
                className="fa fa-code-fork fa-3x "
              />
            </div>,
            <div
              className="pure-u-sm-1 pure-u-md-3-4 block-right "
            >
              <b>
                Want to take a look into the code?
              </b>
              <br />
              <span>
                Browse the repository of this site!
              </span>
            </div>,
          ],
          "className": "pure-u-1 container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <i
                className="fa fa-code-fork fa-3x "
              />,
              "className": "pure-u-sm-1 pure-u-md-1-4 block-left",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fa fa-code-fork fa-3x ",
              },
              "ref": null,
              "rendered": null,
              "type": "i",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <b>
                  Want to take a look into the code?
                </b>,
                <br />,
                <span>
                  Browse the repository of this site!
                </span>,
              ],
              "className": "pure-u-sm-1 pure-u-md-3-4 block-right ",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Want to take a look into the code?",
                },
                "ref": null,
                "rendered": "Want to take a look into the code?",
                "type": "b",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Browse the repository of this site!",
                },
                "ref": null,
                "rendered": "Browse the repository of this site!",
                "type": "span",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "rules": Array [
            "
  font-size: 0.8em;
  cursor: pointer;

  b,
  span,
  i {
    color: #ccc;
    line-height: 1.6em;
  }

  .block-right {
    margin: 12px 0;
  }

  &:hover {
    b,
    span,
    i {
      color: #000;
    }
  }

  @media (max-width: 768px) {
    .container {
      width: 50%;
      margin: 0 25%;
    }

    .block-left,
    .block-right {
      text-align: center;
    }
  }
",
          ],
        },
        "displayName": "styled.div",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "div",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
